CHANGELOG.md

```diff
@@ -4,6 +4,13 @@ NativeScript CLI Changelog
 1.3.0 (2015, September 16)
 ==
 
+### Breaking
+
+`library add` command is deprecated and will be removed completely in one of our next releases (currently scheduled for 1.5).
+You cannot create NativeScript plugins with Android native code using Eclipse projects. You need to import your Eclipse project into Android Studio, convert it to Gradle build and consume the produced AAR file.
+You cannot use Apache Ant to build your projects for Android. Starting with NativeScript 1.3, Android builds require Gradle. Run `tns doctor` on the command line to learn more.
```

Comment:
The users are still able to build their Ant based projects but they will be not able to create Ant projects.